react/PlaceDetails.tsx

```diff
@@ -40,31 +43,37 @@ const PlaceDetails: React.FC<Props> = ({
         .map((line) => {
           return line.filter(
             (fragment) =>
-              address[fragment.name] && !hiddenFields.includes(fragment.name)
+              address[fragment.name as FieldName] &&
+              !hiddenFields.includes(fragment.name as FieldName)
           )
         })
         .filter((line) => line.length > 0)
         .map((line, displayIndex) => (
           <div className="flex" key={displayIndex}>
             {line.map((fragment, index) => {
+              if (typeof index !== 'number') {
+                return
+              }
```

Comment:
I don't know why this is needed... index is clearly a `number` already, but eslint complains with [this](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/restrict-plus-operands.md) anyway.